arista-netdevops-community/network-test-automation
# Support for name and port in inventory

- feat(loader): create hierarchical tests definitions with a catalog loader (#63)
- refactor: remove hack from anta/tests/__init__.py
- ci: remove newline
- refactor: tests/hardware.py (#72)
- refactor(tests): Update VXLAN tests to support new structures (#73)
- refactor(tests): Update SYSTEM tests to support new structures (#74)
- refactor(tests): Update SOFTWARE tests to support new structures (#75)
- refactor(tests): Update PROFILES tests to support new structures (#76)
- refactor(tests): Update MULTICAST tests to support new structures (#77)
- refactor(tests): update interfaces tests to support new structures (#78)
- refactor(tests): Update ROUTING/OSPF tests to support new structures (#79)
- refactor(tests): update mlag tests to support new structures (#80)
- refactor(tests): Update ROUTING/GENERIC tests to support new structures (#81)
- refactor(tests): update bgp tests to support new structures (#82)
- refactor: use TestResult in  tests/configuration (#71)
- test: skip legacy tests (#83)
- add a script to test the tests functions
- add a script to test the tests functions
- feat(result-manager): Implement helpers to set status (#84)
- refactor: cleaning docstrings and code post pr84 (#87)
- ci(pytest): Update inventory.models and implement result_manager.models (#85)
- feat(reporter): Implement initial reporter using rich (#88)
- make(anta-tester): Optimize test execution with search_ip and search_test
- make: Update gitignore for custom folder
- fix(anta): Update docstring for models
- fix(anta): Update docstring
- make(anta-tester): Update CLI options with enable_password and timeout
- fix(tests): Verify ntp timeout and add logging (#91)
- ci: use tox for linting, type checking and testing (#89)
- fix(inventory): Update incorrect typing in InventoryDevices (#92)
- Refactor hardware test (#97)
- make(anta-tester): Update loggging for tests.* modules
- fix(decorators): Update skip_on_platforms to report caller function name (#99)
- fix(tests): fix hardware test for temperature transceivers (#100)
- make(anta-tester): Implement --hostip and --test inputs
- refactor: refactor software.py tests (#93)
- refactor: refactor vxlan tests (#101)
- refactor(tests): Update system with logging & test report update (#94)
- refactor(tests): Update configuration to use logging (#95)
- refactor(tests): Update MLAG tests with logging (#98)
- refactor(tests): Update profiles to use logging and ignore virtual eos (#96)
- refactor(tests): refactor multicast tests (#103)
- refactor(tests): Update routing.bgp tests with logging (#104)
- refactor(tests): add vEOS-lab to skipped platforms (#102)
- refactor(tests): Update routing.generic with logging (#105)
- feat(result-manager): Implement logging for debug purposes (#90)
